### DISCUSSION TITLE
16 ottobre 2023 - esercizio 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#49] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/e401c496-3cd7-4969-84fc-05d55e51ab8d)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/268b947e-6d7f-4e88-b789-b2802c4bb101)

**Comment by IonutCicio** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/2be06075-4315-4eed-850b-d9595e7225b5)

**Comment by niccolo-fato** [comment]
![IMAGE 2024-01-14 11:09:19](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/383452bd-9d04-4ae6-b175-0d1606ea632d)
